build.gradle

```diff
@@ -86,6 +82,12 @@ subprojects {
     sourceCompatibility = 1.8
     [compileJava, compileTestJava]*.options*.encoding = 'UTF-8'
 
+    ktlint {
+        enableExperimentalRules = false
+        verbose = true
+        disabledRules = ["parameter-list-wrapping", "indent", "colon-spacing", "import-ordering"]
```

Comment:
Is it a change or did you have to disable these rules to make new version of ktlint pass?

Reply:
I had to enable them to make ktlint pass, because the previous version was old without those rules. I think these are sensible, but I didn't want to make those changes in this PR, we can ticket it and fix them in a next PR

Reply:
OK, I didn't know that this is not in fact a change compared to current rules. LGTM